walkthroughs/howto-mutual-tls-file-provided/README.md

```diff
@@ -258,6 +243,8 @@ As mentioned above, by default a client proxy will be configured to accept any c
 ./mesh/mesh.sh update_2_client_policy_bad_san
 ```
 
+> Note: if you get an "Unknown parameter" error, you may need to update your AWS CLI
```

Comment:
Good call. I might just as well make this a pre-requisite so they know they're on the latest CLI.